data/tests/html/aria/landmarks-aria.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>ARIA landmarks test suite</title>
+  <style type="text/css">
+    div { border: 1px solid darkblue; padding: 10px }
+  </style>
+</head>
+
+<body>
+  <div role="banner">
+    <p>This is the start of the banner role container</p>
+    <p><a href="/">A11ySupport home</a></p>
+    <div role="search">
+      <p>This is the start of a search role container</p>
+      <label for="txtSearch">Search A11ySupport</label>
+      <input id="txtSearch" type="search" />
+      <p>This is the end of a search role container</p>
+    </div>
+    <p>This is the end of the banner role container</p>
+  </div>
+  <div role="navigation" aria-label="Site sections">
+    <p>This is the start of a named navigation role container</p>
+    <p><button type="button" onclick="alert('nav button 1 pressed');">Nav button 1</button></p>
+    <ul>
+      <li>
+        <a href="/tests">All tests</a>
+      </li>
+      <li>
+        <a href="/faq">FAQ</a>
+      </li>
+      <li>
+        <a href="/contribute">Contribute</a>
+      </li>
+      <li>
+        <a href="/learn">Learn</a>
+      </li>
+    </ul>
+    <p><button type="button" onclick="alert('nav button 2 pressed');">Nav button 2</button></p>
+    <p>This is the end of a named navigation role container</p>
+  </div>
+  <div role="main">
+    <p>This is the start of the main role container</p>
+    <h1>ARIA landmarks testing</h1>
+    <button type="button" onclick="alert('You don\'t follow instructions very well');">Don't click me</button>
+    <p>This is the end of the main role container</p>
+  </div>
+  <div role="complementary">
+    <p>This is the start of the complementary role container</p>
```

Comment:
Do you think each landmark should have a focusable element so that we can test if the role is conveyed when crossing the boundary via `tab`?

Reply:
That was definitely my plan, to continue fleshing out this test page with "stuff" as I work through the ARIA landmarks (and a corresponding page for the HTML landmarks).
Just wanted to concentrate on navigation first and get it right before I committed too much to it.

Reply:
That's totally fine.